ruma-events/src/room/aliases.rs

```diff
@@ -29,15 +29,12 @@ impl AliasesEventContent {
     }
 
     /// Redact an `AliasesEventContent` according to current Matrix spec.
-    pub fn redact(self) -> RedactedAliasesEventContent {
-        RedactedAliasesEventContent { aliases: None }
-    }
-
-    /// Redact an `AliasesEventContent` according to version 1 of the Matrix spec.
-    ///
-    /// This leaves the `aliases` information intact after redaction.
-    pub fn redact_v1(self) -> RedactedAliasesEventContent {
-        RedactedAliasesEventContent { aliases: Some(self.aliases) }
+    pub fn redact(self, version: RoomVersionId) -> RedactedAliasesEventContent {
+        if version.is_version_6() {
```

Comment:
I wonder how we should be handling custom room versions. I thought I had read somewhere that they should generally be considered following the latest room version rules, so this would have to include `|| version.is_custom()`. But I can't find that. Will ask #matrix-spec. But we should still invert this logic so when a new version is introduced, this doesn't revert back to v1 behaviour for that version. Since it's then going to be easier to have custom == v6 logic, let's do that.